Documentation: install from sources (#309)
recognai/biome-text: README.md

</h3>

## Quick Links
- [Documentation](https://www.recogn.ai/biome-text/documentation/)

## Features

> * State-of-the-art and not so state-of-the-art models trained with **your own data** with a simple workflow.

Reviewer: ```suggestion
* State-of-the-art and not so state-of-the-art models trained with **your own data** with simple workflows.
```

> * **Compatible with AllenNLP and Huggingface Transformers**

Reviewer: ```suggestion
* **Compatible with AllenNLP**
```

## Installation
You can install biome.text with pip or from source.

### Pip

> The recommended way of installing the library is using pip. You can install everything required for the library:

Reviewer: ```suggestion
The recommended way for installing the library is using pip. You can install everything required for the library as follows:
```

````shell
git clone https://github.com/recognai/biome-text.git
cd biome-text
python -m pip install .[testing]
````

> Then you must build statics web resources:

Reviewer: ```suggestion
Then you must build static web resources for the explore UI to work:
```

````shell script
cd ui 
npm install 
npm run build
````

*Note: node>=12 is required in your machine. 

> You can follow installation instructions [here](https://nodejs.org/en/download/)*

Reviewer: ```suggestion
You can follow the installation instructions [here](https://nodejs.org/en/download/)*
```

If the `make` command is enabled in your system, you can use `make dev` directive: 

````shell
make dev
````

> And `make ui` for build statics web resources:

Reviewer: ```suggestion
And `make ui` for building static web resources needed for the explore UI method to work:
```

````shell script
make ui
````

> You can see all defined directives as follow:

Reviewer: ```suggestion
You can see all defined directives as follows:
```
